components/micro-gateway-core/src/main/ballerina/src/gateway/utils/utils.bal

```diff
@@ -907,11 +911,15 @@ function readMultipleJWTIssuers() {
         printDebug(KEY_UTILS, "Found new multiple JWT issuer configs");
         string trustStorePath = getConfigValue(LISTENER_CONF_INSTANCE_ID, TRUST_STORE_PATH, DEFAULT_TRUST_STORE_PATH);
         string trustStorePassword = getConfigValue(LISTENER_CONF_INSTANCE_ID, TRUST_STORE_PASSWORD, DEFAULT_TRUST_STORE_PASSWORD);
+        int timestampSkew = getConfigIntValue(SERVER_CONF_ID, SERVER_TIMESTAMP_SKEW, DEFAULT_SERVER_TIMESTAMP_SKEW);
```

Comment:
Can't we pass this from `initAuthHandlers` to here? without reading the same config twise in the same flow?